classes/User.php

```diff
@@ -56,6 +56,7 @@ public function getUserDetails($user_id = False) {
     }
 
     public function getUpdateDetails($this_page, $is_facebook_user = False) {
+        global $THEUSER;
```

Comment:
Can we avoid a `global` here and pass `$THEUSER` in from wherever the function is called? It looks like it's only used in `www/docs/user/index.php`, where `$THEUSER` is in scope already so no need for massive refactoring to inject it all the way down.